### PR TITLE
[12.0] [FIX] Add check on note and section lines

### LINKED ISSUE #1341: Righe "nota" e "sezione" compaiono nell'XML generando errore SDI.
Versioni coinvolte: 12.0

Passi per riprodurre:

Creare una fattura/nota di credito.
Aggiungere una riga di tipo "nota" e una di tipo "sezione".

immagine

Esportare l'xml della fattura elettronica.

Comportamento osservato:

Nell'XML generato vengono aggiunte due righe con le descrizioni indicate in "nota" e "sezione" con importi e quantità pari a zero e aliquota pari a quella predefinita del sistema.

immagine
immagine

Se l'xml viene inviato, SDI lo scarta come "Errore trasmissione" con messaggio:
 
"2.2.2 DatiRiepilogo non presente in corrispondenza di almeno un valore di 2.1.1.7.5 AliquotaIVA o 2.2.1.12 AliquotaIVA"

 
Comportamento atteso:

L'aggiunta di righe per note o per sezioni non dovrebbero generare righe nell'XML.

### PR DESCRIPTION
**Pull request by primes2h**
Fix #1341 

--
Confermo di aver firmato il CLA https://odoo-community.org/page/cla e di aver letto le linee guida su https://odoo-community.org/page/contributing

### CODE REVIEW

File: l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py [hunk 1]
```diff
@@ -616,9 +617,10 @@ def setDettaglioLinee(self, invoice, body):
         if uom_precision < 2:
             uom_precision = 2
         for line in invoice.invoice_line_ids:
-            self.setDettaglioLinea(
-                line_no, line, body, price_precision, uom_precision)
-            line_no += 1
+            if not line.display_type:
```
Comment (eLBati) [inline l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py]: https://github.com/OCA/l10n-italy/issues/1341#issuecomment-514065428

**Comment by eLBati** [comment]
https://github.com/OCA/l10n-italy/issues/1341#issuecomment-514112928

**Comment by eLBati** [comment]
Chiudo questa in favore della https://github.com/OCA/l10n-italy/pull/1440 ?

**Comment by primes2h** [comment]
> Chiudo questa in favore della #1440 ?

Direi di si.

**Pull request closed without merging**